src/App.js

```diff
@@ -16,53 +15,71 @@ class App extends Component {
   constructor() {
     super();
     this.state = {
-      movies: movieData.movies,
-      movie: ''
+      movies: [],
+      movie: '',
+      error: false,
+      videos: [],
     }
   }
 
-  addMovies = () => {
-    this.setState({ movies: [...movieData.movies] })
+  componentDidMount = () => {
+    fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+    .then(response => response.json())
+    .then(data => this.setState({movies : data.movies}))
+    .catch(err => console.log(err))
   }
 
   backToMain = () => {
     this.setState({ movie: ''})
+    this.setState({ videos: []})
+    this.componentDidMount();
+  }
+
+  handleError = (err) => {
+    this.setState({ error: true })
+  }
+
+  playTrailer = (id) => {
+
+    fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}/videos`)
+      .then(response => response.json())
+      .then(data => this.setState({videos : data.videos}))
+      .catch(err => console.log(err))
```

Comment:
Same thing with this .catch... need to make it do something other than console.log()